registry/registry_db.go

```diff
@@ -270,18 +354,139 @@ func dbGetPrefixJwks(prefix string, approvalRequired bool) (*jwk.Set, error) {
 	return &set, nil
 }
 
+func getNamespaceById(id int) (*Namespace, error) {
+	if id < 1 {
+		return nil, errors.New("Invalid id. id must be a positive number")
+	}
+	ns := &Namespace{}
+	adminMetadataStr := ""
+	query := `SELECT * FROM namespace WHERE id = ?`
```

Comment:
Nitpick (I know we do this wrong elsewhere, just trying to make future changes simpler):

Instead of doing `SELECT *`, we should name the columns we are selecting from the database.  This way, if we ever do schema evolution and add columns to this table in the future, we don't break the query.